### PR TITLE
Add additional tests for `fastAggregateVerify`

### LINKED ISSUE #8242: Add additional test for FastAggregateVerify
### Description

According to [PR](https://github.com/LiskHQ/lips/pull/289), we should add additional test to verify zero bytes public key

### Acceptance Criteria

Add additional tests for `FastAggregateVerify` according to https://github.com/LiskHQ/lips/pull/289

### PR DESCRIPTION
**Pull request by bobanm**
### What was the problem?

This PR resolves #8242

### How was it solved?

Added 3 new fixtures, as specified in the [LIP-0038](https://github.com/LiskHQ/lips/pull/289/files)

### How was it tested?

The 3 new test cases are passing 👌🏻

### CODE REVIEW

File: elements/lisk-cryptography/test/bls_lib/lib.spec.ts [hunk 1]
```diff
@@ -144,6 +144,46 @@ describe('bls_lib', () => {
 	});
 
 	describe('blsFastAggregateVerify', () => {
+		describe('special invalid public key', () => {
```
Comment (shuse2) [inline elements/lisk-cryptography/test/bls_lib/lib.spec.ts:147]: instead of adding test manually, I think you can add fixture and include it in the test below?

**Review by AndreasKendziorra — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]
